packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json

```diff
@@ -70,33 +70,34 @@
   "ssi_information_request_page_form_contact_first_name_label": "Contact voornaam",
   "ssi_information_request_page_form_contact_last_name_label": "Contact achternaam",
   "ssi_information_request_page_form_vat_number_label": "BTW identificatienummer",
-  "coc_sharing_data_right_pane_title": "Deel uw gegevens",
+  "coc_sharing_data_right_pane_title": "Deel je gegevens",
   "coc_sharing_data_right_pane_top_paragraph": "Bijna klaar! Verstrek alstublieft ontbrekende gegevens indien nodig.",
-  "coc_credential_verify_request_right_pane_bottom_paragraph": "Scan deze QR-code met de QR-scanner van uw wallet. Als u al op uw telefoon bent, open dan uw wallet door op de knop 'Openen in wallet' te klikken.",
+  "coc_credential_verify_request_right_pane_bottom_paragraph": "Scan deze QR-code met de QR-scanner van je wallet. Als je al op je telefoon bent, open dan je wallet door op de knop 'Openen in wallet' te klikken.",
   "ssi_credentials_landing_page_title": "Ontvang credentials",
   "ssi_credentials_landing_page_text": "Je kunt de onderstaande credentials in je wallet ontvangen. Selecteer een credential en volg de vereiste stappen.",
   "ssi_download_app_button": "Probeer Banqup gratis",
-  "ssi_information_request_page_form_terms_and_conditions_url_text": "terms & conditions",
-  "ssi_welcome_label": "Welcome to Banqup",
+  "ssi_information_request_page_form_terms_and_conditions_url_text": "algemene voorwaarden",
+  "ssi_welcome_label": "Welkom bij Banqup",
   "label_share": "Delen",
   "label_next": "Volgende",
   "label_continue": "Doorgaan",
   "label_finish": "Finish",
```

Comment:
this one is still in english

Reply:
Missed Finish indeed
I left "Download xxx" because that's so common to say in Dutch

Reply:
(fixed)